app/resources/lang/en/voting-lists.php

```diff
@@ -12,8 +12,27 @@
     'against' => 'Against',
     'abstentions' => 'Abstentions',
 
+    'members' => [
+        'title' => 'Members',
+        'search-placeholder' => 'Search by name or country',
+        'show-more' => 'Show all MEPs',
+    ],
+
     'groups' => [
+        'title' => 'Political Groups',
         'count' => ':voted of :total MEPs voted',
     ],
 
+    'countries' => [
+        'title' => 'Countries',
+        'search-placeholder' => 'Search by name',
+        'show-more' => 'Show all countries',
+    ],
+
+    'download' => [
+        'heading' => 'Open Data',
+        'text' => 'We provide raw voting data, ready to use in your own analyses, visualizations or applications. Data is provided in CSV format and licensed under Creative Commons Attribution License 4.0.',
```

Comment:
```suggestion
        'text' => 'We provide raw data for this vote, ready to use in your own analyses, visualizations or applications. Data is provided in CSV format and licensed under Creative Commons Attribution License 4.0.',
```
Fände es gut, wenn aus dem Text hier nochmal ersichtlich wird, dass es es jeweils Daten **einer** Abstimmung sind.
So ist aber auch nicht toll, weil es dann finde ich so klingt als ob es das nicht immer gäbe.